ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/monitor-sessions.j2

```diff
@@ -0,0 +1,10 @@
+{# eos - monitor sessions configuration #}
```

Comment:
I think this slipped into the wrong PR ?

Reply:
yup argh, i'm also adding support for span config in a different pr.. let me remove it here, thanks for catching!